tgui/packages/tgui/interfaces/IssueReporter.js

```diff
@@ -0,0 +1,226 @@
+        break;
+  }
+
+  return (
+    <Window
+      width={340}
+      height={windowHeight}
+      overflow="auto"
+      theme="generic"
+      title={"Issue reporter " + stage + "/3"}>
+      <Window.Content>
+        <Flex direction="column" height="100%">
+          <Flex.Item
+            grow={1}>
+            {content}
+          </Flex.Item>
+          <Flex.Item
+            alignSelf="flex-end">
+            <Section>
+              <Flex direction="row" width="100%">
+                {stage !== 1 ? (
+                <Flex.Item
+                  grow={1}>
+                  <Button
+                    fontSize={1.5}
+                    onClick={() => {
+                      if (stage === 1)
+                      {
+                        return;
+                      }
+                      setStage(stage - 1);
+                    }}>
+                    Previous
+                  </Button>
+                </Flex.Item>) : (
+                  <Flex.Item grow={1} />
+                )}
+                <Flex.Item
+                  alignSelf="flex-end">
+                  <Button
+                    fontSize={1.5}
+                    onClick={() => {
+                      if (stage === 3)
+                      {
+                        // Submit the report and close
+                        act('submit', {
+                          title: issueName,
+                          expected: expected,
+                          actual: actual,
+                          replicationSteps: replicationSteps,
+                          isRegression: isRegression,
+                        });
+                        return;
+                      }
+                      setStage(stage + 1);
+                    }}>
+                    {stage === 3 ? "Submit in browser" : "Continue"}
+                  </Button>
+                </Flex.Item>
+              </Flex>
+            </Section>
+          </Flex.Item>
+        </Flex>
+      </Window.Content>
+    </Window>
+  );
+};
+
+const Regression = (props, context) => {
+
+  const {
+    isRegression,
+    setIsRegression,
+    issueName,
+    setIssueName,
+  } = props;
+
+  return (
+    <>
+      <Section title="Issue Title">
+        Please enter a concise name for the issue
+        <Input
+          mt={1}
+          width="100%"
+          value={issueName}
+          onInput={(e, text) => setIssueName(text)} />
+      </Section>
+      <Section title="Issue Type">
+        Was the issue being reported previously functioning as intended?
+        <i>(Please select &#39;No&#39; if unknown).</i>
+        <Box
+          mt={1}
+          mb={1}
+          width="100%">
+          <ButtonCheckbox
+            overflowX="wrap"
+            ml={1}
+            style={{ 'font-weight': 'normal', 'font-size': '12px' }}
+            content={isRegression
+              ? "Yes - This is a new issue"
+              : "No - This hasn't been known to work"}
```

Comment:
This checkbox is bad UI. Use two checkboxes, disable the active one.

Reply:
That makes no sense, although I agree that the wording could be better.

Reply:
It should act like radio buttons. The best way to recreate it is using two checkboxes and then disabling the selected one.